app/models/ganja.rb

```diff
@@ -1,3 +1,12 @@
 class Ganja < ApplicationRecord
   belongs_to :user
+
+  validates :name, presence: true
+  validates :strain, presence: true
+  validates :unit_price, presence: true
+  validates :pickup_local, presence: true
+  validates :variety, inclusion: { in: %w[edibles Edibles
```

Comment:
change this to be either edible or flower, we need to do a lot of if else statements with this and it wont be good if there are many